deck/forms.py

```diff
@@ -30,6 +30,11 @@ class Meta:
                 'class': 'inline-input',
                 'placeholder': 'Due Date'
             }),
+            'event_date': CustomDateTimeWidget(attrs={
+                'id': 'id_event_date',
+                'class': 'inline-input',
+                'placeholder': 'Event Date'
```

Comment:
Can you please add the `gettext` in the  `'Event Date'` and  `'Due Date'`?

Reply:
sure